plugins/janus_sip.c

```diff
@@ -1100,6 +1104,32 @@ static void janus_sip_random_string(int length, char *buffer) {
 	}
 }
 
+static void parse_custom_headers(json_t *root, char *custom_headers) {
```

Comment:
Can you prefix this function name with `janus_sip_`? It's custom in all the Janus code.

Reply:
Done.